recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,5 @@
+# Please consult conda-forge/core before doing this
+MACOSX_DEPLOYMENT_TARGET:  # [osx and x86_64]
+  - "10.13"                # [osx and x86_64]
+MACOSX_SDK_VERSION:        # [osx and x86_64]
+  - "10.13"                # [osx and x86_64]
```

Comment:
These variables are also set in ...
https://github.com/conda-forge/scipoptsuite-feedstock/blob/c2c8639b7a63f6abc5db359bc1dcbca48d8d6102/.ci_support/osx_64_.yaml#L1-L4
and ...
https://github.com/conda-forge/scipoptsuite-feedstock/blob/c2c8639b7a63f6abc5db359bc1dcbca48d8d6102/.ci_support/osx_arm64_.yaml#L1-L2

Reply:
@dg0yt yes that is normal. The files in `.ci_support` are some scripts files and utility used in CI.
They are not meant to be edited directly by the developers, instead they are regenerated during "render".

The `osx_64` file is the result of this render. The `osx_arm64` was left unchanged during (from the `[osx and x64]` selector). It has higher version because M1 did not appear before `11`, so no need to target anything lower.